### PR TITLE
Delete favorite list detail api

### PR DESCRIPTION
**Pull request by wuhanwei**
request body : 
{
    "member_account":"a123454@gmail.com.tw",
    "favorite_list_name":"etf",
    "stock_id":"00892"
}

### CODE REVIEW

File: demo/src/main/java/com/example/demo/controller/MemberController.java [hunk 1]
```diff
@@ -20,6 +21,16 @@ public class MemberController {
     @Autowired
     MemberService memberService;
 
+    @PostMapping("/member/deleteFavoriteListDetail")
```
Comment (Timmy823) [inline demo/src/main/java/com/example/demo/controller/MemberController.java]: deleteFavoriteListStock 沒人知道你的detail是什麼

File: demo/src/main/java/com/example/demo/controller/MemberController.java [hunk 2]
```diff
@@ -20,6 +21,16 @@ public class MemberController {
...
+    @CrossOrigin(origins = "*", allowedHeaders = "*")
+    public JSONObject deleteFavoriteListDetail(@Valid @RequestBody FavoriteListDetailDeleteParam input) {
```
Comment (Timmy823) [inline demo/src/main/java/com/example/demo/controller/MemberController.java]: naming

File: demo/src/main/java/com/example/demo/Component/MemberComponent/FavoriteListDetailDeleteParam.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import lombok.Data;
+
+@Data
+public class FavoriteListDetailDeleteParam {
+    @NotEmpty(message = "member_account不可為空")
```
Comment (Timmy823) [inline demo/src/main/java/com/example/demo/Component/MemberComponent/FavoriteListDetailDeleteParam.java]: NotNull也加一下

File: demo/src/main/java/com/example/demo/Component/MemberComponent/FavoriteListDetailDeleteParam.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    @NotEmpty(message = "stock_id不可為空")
+    private String stock_id;
+
+    public FavoriteListDetailDeleteParam(String member_account, String favorite_list_name, String stock_id) {
+        super();
+        this.account = member_account;
+        this.list_name = favorite_list_name;
+        this.stock_id = stock_id;
+    }
+}
```
Comment (Timmy823) [inline demo/src/main/java/com/example/demo/Component/MemberComponent/FavoriteListDetailDeleteParam.java]: 空行

File: demo/src/main/java/com/example/demo/Entity/FavoriteListDetailModel.java [hunk 5]
```diff
@@ -25,7 +25,7 @@ public class FavoriteListDetailModel {
     private Long list_detail_id;
 
     @Column(name = "list_name_id", nullable = false, columnDefinition = "BINARY(36)")
-    private String list_name_id;
+    private Long list_name_id;
```
Comment (Timmy823) [inline demo/src/main/java/com/example/demo/Entity/FavoriteListDetailModel.java:28]: 為何要改

Comment (wuhanwei) [reply demo/src/main/java/com/example/demo/Entity/FavoriteListDetailModel.java:28]: > 為何要改

list_nam_id is generate  identity value in favorite list name model.

**Review by Timmy823 — APPROVED** [review]

**Pull request merged**